File: app/src/main/java/net/sytes/kai_soft/letsbuyka/Lists/List.java
package net.sytes.kai_soft.letsbuyka.Lists;

import java.io.Serializable;

/**
 * Created by Лунтя on 06.06.2018.
 */

public class List implements Serializable {
    private int id;
    private String itemName;

    public List(int id, String itemName){
        this.id = id;
        this.itemName = itemName;
    }

    public int getId(){
        return this.id;
    }

    public String getItemName(){
        return this.itemName;
    }
    public void setItemName(String itemName){
        this.itemName = itemName;
    }

    @Override
    public String toString() {
        return getId() + ", " + getItemName();
    }
}

File: app/src/main/java/net/sytes/kai_soft/letsbuyka/Lists/List.kt
package net.sytes.kai_soft.letsbuyka.Lists

import android.util.Log
import java.io.Serializable

class List(val id: Int, var itemName: String): Serializable{

    companion object {
        private const val myTag = "letsbuy_List"
    }

    override fun toString(): String {
        Log.i(myTag, "toString()")
        return "$id, $itemName"
    }
}